storybook/index.js

```diff
@@ -0,0 +1,30 @@
+// if you use expo remove this line
+import { AppRegistry } from 'react-native';
+
+import { getStorybookUI, configure, addDecorator } from '@storybook/react-native';
+import { withKnobs } from '@storybook/addon-knobs';
+
+import { loadStories } from './storyLoader';
+
+import './rn-addons';
+
+// enables knobs for all stories
+addDecorator(withKnobs);
+
+// import stories
+configure(() => {
+	require('./GettingStarted.stories');
+	loadStories();
+}, module);
+
+// Refer to https://github.com/storybookjs/storybook/tree/master/app/react-native#start-command-parameters
+// To find allowed options for getStorybookUI
+const StorybookUIRoot = getStorybookUI({
+	asyncStorage: null,
+});
+
+// If you are using React Native vanilla and after installation you don't see your app name here, write it manually.
+// If you use Expo you should remove this line.
+AppRegistry.registerComponent('MetaMask', () => StorybookUIRoot);
```

Comment:
Since we are still relying on a manual change, I'd suggest avoid registering the storybook root in this file and just toggle comments in `/index.js`
```suggestion
// AppRegistry.registerComponent('MetaMask', () => StorybookUIRoot);
```
And then in [`/index.js`](https://github.com/MetaMask/metamask-mobile/blob/12549c2f34af7d09f31f1b9f9bb3ab1a7d61cb1f/index.js#L64-L67)

```diff
+/* Uncomment and comment regular registration below */
+// import Storybook from './storybook';
+// AppRegistry.registerComponent(name, () => Storybook);
+
/**
 * Application entry point responsible for registering root component
 */
AppRegistry.registerComponent(name, () => Root);
```

Reply:
Have updated `index.js` and readme docs [1656614](https://github.com/MetaMask/metamask-mobile/pull/3374/commits/165661424ac570fa44d7414f760913d01f8598d6)

Reply:
Have updated `index.js` and readme docs [1656614](https://github.com/MetaMask/metamask-mobile/pull/3374/commits/165661424ac570fa44d7414f760913d01f8598d6)

Reply:
Have updated `index.js` and readme docs [1656614](https://github.com/MetaMask/metamask-mobile/pull/3374/commits/165661424ac570fa44d7414f760913d01f8598d6)